src/reference/modules/usage/pages/advanced_configuration_management.adoc

```diff
@@ -284,48 +284,27 @@ userManagement/7.0: Role: [user-role] (with login [login])
 ----
 
 
-== Directives ordering
+== Policies ordering
 
-Configuration in Rudder are based on desired states, describing the expected state of the system. However, there are cases where having order is desirable (like ensuring that a JVM is present before deploying an Application server, or ensuring a user is present before setting it sudoers), even if it will converge over the course of several agent runs.
+Configuration in Rudder is based on desired states, describing the expected state of the system, and not a an ordered list of actions to run.
+However, there are cases where having order is desirable (like ensuring that a JVM is present before deploying an Application server, or ensuring a user is present before setting it sudoers), even if it will converge over the course of several agent runs.
 
-In Rudder, there is two separated ways to order things, depending the type of Technique". So, before that, we need to explain how Policies are generated on the
-agent from Directives based on the same Technique.
+Rudder sorts the policies following this (best-effort) principle:
 
-=== Policy generation and Directive merge
+* Rules are executed in alphanumeric order
+* Inside of a rule, directives are executed in alphanumeric order, except for:
 
-In Rudder, policies are generated from directives. In the general case, each directive will lead to one policy on the node. 
+** *Mono-instance* techniques: only one directive is selected and others are discarded (and they will be tagged as such in node detail compliance) 
 
-There are two exceptions to that rule:
+*** the directive with the highest priority will be selected
+*** in case of equal priorities, the rule name and then the directive name are used for sorting them (in alphanumeric order)
 
-- for unique (non multi-instance) technique, the directive with the highest priority is selected and others are discarded (and they will be tagged as such in node detail compliance),
-- for multi-instance technique, if the technique `POLICYGENERATION` metadata tag has value `merged`, then all directives will be merged into *one* policy on the node. That behavior used to be the default one until Rudder 4.3. See https://issues.rudder.io/issues/10625[Don't merge directive from same technique on generation] for more information about it.
+** *Limited multi-instance* techniques: all directives will be merged into *one* policy on the node
```

Comment:
```suggestion
** *Limited multi-instance* techniques: all directives will be merged into *one* policy on the node, and thus be applied on the first instance of directives
```

Reply:
I'm not sure about what you mean, isn't it the same as the "the order of the merged policy itself will be based in the rule and directive names of its most prioritary source directive" line?

Reply:
ping @ncharles